flink-formats/flink-csv/src/test/java/org/apache/flink/formats/csv/CsvFormatFilesystemStatisticsReportTest.java

```diff
@@ -34,16 +34,16 @@
 /**
  * Test for statistics functionality in {@link CsvFormatFactory} in the case of file system source.
  */
-public class CsvFormatFilesystemStatisticsReportTest extends CsvFormatStatisticsReportTest {
+class CsvFormatFilesystemStatisticsReportTest extends CsvFormatStatisticsReportTest {
 
+    @Override
     @BeforeEach
     public void setup(@TempDir File file) throws Exception {
```

Comment:
nit: since the hierarchy is in different packages package private does not work here however probably it should either changed to `protected` for the whole hierarchy or marked like `@VisibleForTestting`

Reply:
I guess this is true, but I'd prefer to avoid `@VisibleForTesting` for classes in the `src/test/java` hierarchy... There are a few places in the Flink code where I've seen this, but it's pretty rare and probably not what the contributor really intended!

I'll change it to protected.